app/routes/changes-enquiries/death/functions.js

```diff
@@ -34,9 +34,10 @@ const [CANNOT_CALCULATE, OVERPAYMENT, ARREARS, NOTHING_OWED, DEATH_NOT_VERIFIED]
 
 function getAddDateDeath(req, res) {
   const { awardDetails } = req.session;
+  const keyDetails = keyDetailsHelper.formatter(awardDetails);
   const details = dataStore.get(req, 'date-of-death', 'death');
   res.render('pages/changes-enquiries/death/enter-date', {
-    awardDetails,
```

Comment:
Why was this removed

Reply:
it didn't appear to be used by the view but I've put it back since it's outside the story